pkg/commands/github.go

```diff
@@ -0,0 +1,50 @@
+package commands
+
+import (
+	"encoding/json"
+
+	"github.com/jesseduffield/lazygit/pkg/commands/models"
+	"github.com/jesseduffield/lazygit/pkg/gui/presentation"
+)
+
+func (c *GitCommand) GithubMostRecentPRs() (map[string]models.GithubPullRequest, error) {
+	commandOutput, err := c.OSCommand.RunCommandWithOutput("gh pr list --limit 50 --state all --json state,url,number,headRefName,headRepositoryOwner")
```

Comment:
Yes, we should add the check.
I thought it would be a good idea to add a flag like `useGhCommand` and check it acoording to that flag in `main.go`, what do you think?

Reply:
thinking out loud about the pros and cons:
pros:
* by requiring the user to explicitly state that they want to use the gh tool, we can now surface errors to them so that e.g. they can ensure that they install gh if it's not installed, or get it to the right version if not already.
* allows us to battle-test the feature against a small set of users if we later on want to make it default
cons:
* it's cool for people who already have gh installed (with the right version) if they magically see PR numbers next time they update lazygit.

Given the above, I reckon adding a flag for this is a good idea